packages/plugin-inline-script-content/inline-script-content.js

```diff
@@ -102,7 +102,7 @@ module.exports = {
       'Notification', 'SVGElementInstance', 'Screen', 'TextTrack', 'TextTrackCue', 'TextTrackList',
       'WebSocket', 'WebSocketWorker', 'Worker', 'XMLHttpRequest', 'XMLHttpRequestEventTarget', 'XMLHttpRequestUpload'
     ], o => {
-      if (!win[o] || !win[o].prototype || !win[o].prototype.hasOwnProperty || !win[o].prototype.hasOwnProperty('addEventListener')) return
+      if (!win[o] || !win[o].prototype || !Object.prototype.hasOwnProperty.call(win[o].prototype, 'addEventListener')) return
```

Comment:
A quick search suggests `Object.prototype.hasOwnProperty.call` is supported and preferred whereas `hasOwnProperty` doesn't exist for "host" objects such as `window`.  Possibly a slight change in behaviour to be aware of.

Reply:
Yeah, I presume this was the reason the linter told me change it!